Assignment5Objects/src/assignment5objects/MainProgram.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * @author Samantha Shurie
+ * Assignment 5: Working with Objects
+ */
+package assignment5objects;
+
+
+public class MainProgram {
+   
+    /**
+     * Main method implements both the RetailItem and Car classes. This method
+     * creates a new instance of the Car class and increases/decreases it's speed
+     * field. It also creates three instances of the RetailItem class and prints
+     * their fields.
+     * @param args 
+     */
+    public static void main(String[] args) {
+        int currentSpeed = 0;   //used to retrieve speed value of car object
```

Comment:
It is risky to constantly reassign a single variable over the course of a program. You should declare the variable as close to where you are go use it. This helps use keep logical units localize and help use implement DRY (Don't Repeat Yourself) more easily.